VultisigApp/VultisigApp/iOS/View/Vault/VaultDetailView+iOS.swift

```diff
@@ -40,7 +40,9 @@ extension VaultDetailView {
     }
     
     var cells: some View {
-        return ForEach(viewModel.groups, id: \.id) { group in
+        return ForEach(viewModel.groups.sorted(by: {
```

Comment:
Groups is sorting inside `categorizeCoins`. Do we need to do it one more time here?